docs/en/sql-reference/functions/url-functions.md

```diff
@@ -78,30 +78,78 @@ topLevelDomain(url)
 
 - `url` — URL. [String](../data-types/string.md).
 
+:::note
 The URL can be specified with or without a scheme. Examples:
 
 ``` text
 svn+ssh://some.svn-hosting.com:80/repo/trunk
 some.svn-hosting.com:80/repo/trunk
 https://clickhouse.com/time/
 ```
+:::
 
 **Returned values**
 
 - Domain name if ClickHouse can parse the input string as a URL. Otherwise, an empty string. [String](../data-types/string.md).
 
 **Example**
 
+Query:
+
 ``` sql
 SELECT topLevelDomain('svn+ssh://www.some.svn-hosting.com:80/repo/trunk');
 ```
 
+Result:
+
 ``` text
 ┌─topLevelDomain('svn+ssh://www.some.svn-hosting.com:80/repo/trunk')─┐
 │ com                                                                │
 └────────────────────────────────────────────────────────────────────┘
 ```
 
+### topLevelDomainRFC
+
+Extracts the the top-level domain from a URL. It is similar to [topLevelDomain](#topleveldomain), but conforms to RFC 3986.
```

Comment:
Side note and not blocking this PR:

The beginning of the document currently contains sentence "All these functions do not follow the RFC.". The average reader will be confused by this.

Someone reading the docs will not really understand how `topLevelDomainRFC` differs from `topLevelDomain`. What is generally missing in the entire md file is the specific non-RFC3986 URL format (with examples) recognized by all functions documented in this file, and likewise the RFC3986 format recognized by the `*RFC` functions described in this file. [Here](https://github.com/ClickHouse/ClickHouse/pull/42274) is the PR which added the RFC variant, perhaps the contained examples are helpful to come up with better documentation (--> other PR).

Reply:
@rschu1ze I've added the other missing `RFC` variants with some examples for which the RFC variant will work but not the non-`RFC` one. I've also made some changes to the beginning of the document to better clarify the difference. Let me know if we need to expand on this further or modify, remove something.

Reply:
Should be good, thanks.

I also went once over the docs and pushed some fixups.